src/instrument.mli

```diff
@@ -0,0 +1,14 @@
+open Import
+
+type pos = Before | After
+
+type t = { transformation :  Parsetree.structure -> Parsetree.structure ; position : pos}
+
+(** Define an instrumentation that can be passed to [Driver.register_transformation] to 
+    instrument an implementation file. [transformation] is the transformation that will
+    be applied to the AST; [position] specifies if it should be applied before or after
+    rewriters defined through [rules], [impl] or [intf] are applied.*)
+val make 
+  : transformation:(Parsetree.structure -> Parsetree.structure) 
```

Comment:
`transformation` feels a tad long here, and when combined with `register_transformation` it feels a bit redundant:

```ocaml
Driver.register_transfirmation ~instrument:(Instrument.make ~transformation ...)
```

Skipping the label entirely seems fine here.

Reply:
Thanks. I've also changed that.